build/mac/cross-compile/README.md

```diff
@@ -0,0 +1,113 @@
+# Cross-compiling Brave
+
+The code in this folder makes it possible to compile Brave for macOS in Linux.
```

Comment:
> makes it possible to compile Brave for macOS in Linux

is it really like that? I thought we can build unsigned brave and tests (mac on linux) even without this change.

Should this mention instead that it makes possible to build the signed `.app` along with valid DMG and PKG?

Reply:
True, I addressed this by modifying the README in 1d5ccac8edc32061ec03e3da0f08bbda9e6b55b5.